src/main/java/com/hearthstone/hearthstone_deck_storage/common/config/ApiClientConfig.java

```diff
@@ -0,0 +1,6 @@
+package com.hearthstone.hearthstone_deck_storage.common.config;
+
+public interface ApiClientConfig {
```

Comment:
remove it if you don't use it now and all other related classes. Add in the next PR.